server/models/offchain_thread.ts

```diff
@@ -147,6 +173,14 @@ export default (
       foreignKey: 'thread_id',
       constraints: false,
     });
+    models.OffchainThread.hasMany(models.LinkedThread, {
```

Comment:
Nope, since they're just references to the joins table. Figured that, to minimize query time, I'd always lazy load linked threads, and a full through association wouldn't serve much purpose. But maybe it'd be nice to have anyway? 

Come to think of it, I wonder if we should do a refactor for threads, such that we lazy load all data that isn't necessary for discussion listing index page, and only fetch the rest when navigating to a discussion thread. Right now, we lazy load version history and linked threads, but eager load body text, collaborators, and some booleans that may not be strictly necessary.